pkg/odo/genericclioptions/resolve.go

```diff
@@ -0,0 +1,128 @@
+package genericclioptions
+
+import (
+	"fmt"
+
+	"github.com/openshift/odo/pkg/envinfo"
+	"github.com/openshift/odo/pkg/odo/util"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ResolveAppFlag resolves the app from the flag
+func ResolveAppFlag(command *cobra.Command) string {
```

Comment:
Doesn’t this make more sense to be in flag.go

Reply:
i will try to remove this in next PR's

Reply:
Ok